packages/application/src/index.ts

```diff
@@ -525,6 +533,31 @@ export class Application<T extends Widget = Widget> {
     return this._delegate.promise;
   }
 
+  /**
+   * Collect the deferred plugins.
+   *
+   * @returns The list of all the deferred plugins.
+   */
+  deferredPlugins(): string[] {
```

Comment:
Let's use a getter here:

```suggestion
   * The list of all the deferred plugins.
   */
  get deferredPlugins(): string[] {
```